client/views/chan.tpl

```diff
@@ -1,7 +1,14 @@
 {{#each channels}}
 <div data-id="{{id}}" data-target="#chan-{{id}}" data-title="{{name}}" class="chan {{type}} chan-{{slugify name}}">
 	<span class="badge{{#if highlight}} highlight{{/if}}">{{#if unread}}{{roundBadgeNumber unread}}{{/if}}</span>
+	{{#equal type "lobby"}}
+		<button class="add-channel" aria-label="Add channel" data-id="{{id}}"></button>
```

Comment:
Could you add a tooltip that says "Join a channel..." here? I think join is more explicit than add, and ellipsis sort of because of [this](https://medium.com/@jsaito/the-mighty-ellipsis-6c2c00ddc864).